command_line/show_mask_info.py

```diff
@@ -0,0 +1,24 @@
+# LIBTBX_SET_DISPATCHER_NAME dev.dxtbx.show_mask_info
+
+import sys
+
+from dxtbx.model.experiment_list import ExperimentListFactory
+
+
+def show_mask_info():
```

Comment:
Passing the input arguments at this point would make the function more useful elsewhere.
```suggestion
def show_mask_info(filenames):
```

Reply:
Also resolved in 01b904ad3fa9ecf840c2fb5f5b35d8a59e9a038a